src/oumi/cli/cli_utils.py

```diff
@@ -137,3 +138,46 @@ def set_log_level(level: Optional[LogLevel]):
         callback=set_log_level,
     ),
 ]
+
+
+def resolve_oumi_prefix(
+    config_path: str, output_dir: Optional[Path] = None
+) -> tuple[str, Path]:
+    """Resolves oumi:// prefix and determines output directory.
+
+    Args:
+        config_path: Path that may contain oumi:// prefix
+        output_dir: Optional output directory override
+
+    Returns:
+        tuple[str, Path]: (cleaned path, output directory)
+    """
+    config_path = config_path[7:]
```

Comment:
```suggestion
    oumi_prefix = "oumi://"
    if config_path.lower().startswith(oumi_prefix):
        config_path = config_path[len(oumi_prefix):]
```

This allows users to use the `oumi://` prefix when fetching, but it's not required.

So they could call `oumi fetch /configs/recipes/some_config.yaml`
or 
`oumi fetch oumi://configs/recipes/some_config.yaml`